presto-parser/src/main/java/com/facebook/presto/sql/tree/WindowName.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql.tree;
+
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static java.util.Objects.requireNonNull;
+
+public class WindowName
```

Comment:
We need a way to express an "alias" in addition to the definition of the window in the AST.
E.g. in query
```sql
SELECT x, sum(x) OVER w
FROM (VALUES (1), (2), (3)) AS t (x)
WINDOW w AS (ORDER BY x);
```
_OVER_ **w** would be `WindowName` (w/o _OVER_ itself)
while **w AS (ORDER BY x)** would be `WindowDefinition` (compound of `Identifier` (**w**) and `WindowSpecification` (**ORDER BY x**).